### PR TITLE
Liquidation module

### PR DESCRIPTION
**Pull request by 0xdomrom**

### CODE REVIEW

File: src/modules/LiquidateModule.sol [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  DutchAuction public auction;
+  ICashAsset public cashAsset;
+
+  constructor(IMatching _matching, DutchAuction _auction) BaseModule(_matching) {
+    auction = _auction;
+    cashAsset = _auction.cash();
+  }
+
+  /**
+   * @notice transfer asset between 2 subAccounts
```
Comment (antoncoding) [inline src/modules/LiquidateModule.sol:37]: the comment is wrong

File: test/modules/Liquidate.t.sol [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+  function testCanSubmitFeedDataWithBid() public {
+    IBaseManager.ManagerData[] memory data = new IBaseManager.ManagerData[](1);
+    data[0] = IBaseManager.ManagerData({receiver: bob, data: ""});
+
+    (IActionVerifier.Action[] memory actions, bytes[] memory signatures, bytes memory actionData) =
+      _getBidActionData(1000e18, 1e18, 0, 0, true, abi.encode(data));
+
+    // Easiest way to test that the call to the receiver happened, since tests aren't setup to support full feeds
+    vm.expectRevert(IBaseManager.BM_UnauthorizedCall.selector);
```
Comment (antoncoding) [inline test/modules/Liquidate.t.sol:93]: nice!

File: src/modules/LiquidateModule.sol [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+// Inherited
+import {IBaseModule} from "../interfaces/IBaseModule.sol";
+import {BaseModule} from "./BaseModule.sol";
+
+// Interfaces
+import {ISubAccounts} from "v2-core/src/interfaces/ISubAccounts.sol";
+import {IManager} from "v2-core/src/interfaces/IManager.sol";
+import {IAsset} from "v2-core/src/interfaces/IAsset.sol";
+import {DutchAuction} from "v2-core/src/liquidation/DutchAuction.sol";
```
Comment (antoncoding) [inline src/modules/LiquidateModule.sol:15]: i just realized we don't have `bid` function defined in `IDutchAuction` 🤣

**Review by antoncoding — APPROVED** [review]
only 1 minor comment!

File: src/modules/LiquidateModule.sol (hunk 1 above)
Comment (0xdomrom) [reply src/modules/LiquidateModule.sol:37]: will merge and fix in followup (rfq)

**Pull request merged**